transactions/config/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <suppress files="^((?!.*test[\\/]java[\\/]org[\\/]ehcache[\\/]transactions[\\/]xa[\\/]docs[\\/].*).)*$" checks="AvoidStaticImport"/>
```

Comment:
For no special reason. I wondered how independent the transactions modules has to be, do I bundled everything including the doc in it. I could move all that to the docs module if you believe this makes more sense.